File: desktop-windows/src/P2PAudio.Windows.Core/Models/ConnectionDiagnostics.cs
namespace P2PAudio.Windows.Core.Models;

public sealed record ConnectionDiagnostics(
    NetworkPathType PathType = NetworkPathType.Unknown,
    int LocalCandidatesCount = 0,
    string SelectedCandidatePairType = "",
    string FailureHint = "",
    FailureCode? NormalizedFailureCode = null
);

File: mobile-android/app/src/main/java/com/example/p2paudio/model/ConnectionDiagnostics.kt
package com.example.p2paudio.model

data class ConnectionDiagnostics(
    val pathType: NetworkPathType = NetworkPathType.UNKNOWN,
    val localCandidatesCount: Int = 0,
    val selectedCandidatePairType: String = "",
    val failureHint: String = ""
)
